WordPress/src/main/java/org/wordpress/android/ui/mysite/QuickStartRepository.kt

```diff
@@ -129,6 +129,18 @@ class QuickStartRepository
         }
     }
 
+    fun skipQuickStart() {
+        selectedSiteRepository.getSelectedSite()?.let { site ->
+            val siteId = site.id.toLong()
```

Comment:
Minor (🔍): As discussed, let's start using `siteLocalId` instead of `siteId` for the variable name, when that is derived from `site.id`.

Reply:
Renamed in ade7915